From d4b145a9ae7f196118b21c36bfc83433cd248b52 Mon Sep 17 00:00:00 2001
From: Dhaval Shah <dhaval.shah@xilinx.com>
Date: Mon, 13 Jan 2020 00:20:51 -0800
Subject: [PATCH 0813/1852] dt-bindings: soc: xilinx: Update VCU bindings for
 vcu reset pin

commit 25611b0644dd5b6b0ee8637fe9ec8b3ac6dae418 from
https://github.com/Xilinx/linux-xlnx.git

Updated VCU binding for reset GPIO pin as optional property

Signed-off-by: Dhaval Shah <dhaval.shah@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Rohit Visavalia <rohit.visavalia@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt b/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
index 014168eb52d0..8928ff224465 100644
--- a/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
+++ b/Documentation/devicetree/bindings/soc/xilinx/xlnx,vcu.txt
@@ -39,6 +39,12 @@ Required properties:
 		- interrupts: interrupt number to the cpu.
 		- interrupt-parent: the phandle for the interrupt controller
 		  that services interrupts for this device.
+
+Optional properties:
+- reset-gpios : The GPIO used to reset the VCU, if available. Need use this
+		reset gpio when in design 'vcu_resetn' is driven by gpio. See
+		Documentation/devicetree/bindings/gpio/gpio.txt for details.
+
 Example:
 
 	xlnx_vcu: vcu@a0040000 {
@@ -48,6 +54,7 @@ Example:
 		reg = <0x0 0xa0040000 0x0 0x1000>,
 			 <0x0 0xa0041000 0x0 0x1000>;
 		reg-names = "vcu_slcr", "logicore";
+		reset-gpios = <&gpio 0x4e GPIO_ACTIVE_HIGH>;
 		#clock-cells = <0x1>;
 		clock-names = "pll_ref", "aclk", "vcu_core_enc", "vcu_core_dec", "vcu_mcu_enc", "vcu_mcu_dec";
 		clocks = <&si570_1>, <&clkc 71>, <&xlnx_vcu 1>, <&xlnx_vcu 2>, <&xlnx_vcu 3>, <&xlnx_vcu 4>;
-- 
2.31.1

